Backend/Data/FavouriteMovieRepository.cs

```diff
@@ -0,0 +1,38 @@
+using Backend.Data.Abstraction;
+using Backend.Models;
+using Backend.Utils;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Data;
+internal class FavouriteMovieRepository : IFavouriteMovieRepository
+{
+    private readonly MyDbContext _context;
+    public FavouriteMovieRepository(MyDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task AddFavourite(Favourite favourite)
+    {
+        await _context.Favourites.AddAsync(favourite);
+        await _context.SaveChangesAsync();
+
+    }
+
+    public async Task DeleteFavourite(Favourite favourite)
```

Comment:
NIT: Remove favorite 🗡️

Reply:
:grey_question:  Do you mean to rename the method to RemoveFavourite()?

Reply:
Yes precisely - but this is just nit picking not something important 🥂